Code/Framework/AzToolsFramework/AzToolsFramework/Prefab/PrefabPublicHandler.cpp

```diff
@@ -330,6 +330,11 @@ namespace AzToolsFramework
                                                  "(PrefabEditorEntityOwnershipInterface unavailable)."));
             }
 
+            if (!prefabEditorEntityOwnershipInterface->GetRootPrefabInstance())
+            {
+                return AZ::Failure(AZStd::string("Could not instantiate prefab - no level prefab is loaded."));
```

Comment:
nit : consider changing this to 'no root prefab is found', just to keep the door open for non-level workflows in the future. Also, provide an additional message like this to make it clear to the user that this is not a bug but expected behavior : "Currently, prefabs can only be instantiated inside a level'